atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -109,6 +113,7 @@ def __init__(self, tasker, workflow, labels,
         self.dont_overwrite = dont_overwrite
         self.dont_overwrite_if_in_dockerfile = dont_overwrite_if_in_dockerfile
         self.aliases = aliases or self.DEFAULT_ALIASES
+        self.info_url_format = info_url_format or DEFAULT_INFO_URL_FORMAT
```

Comment:
I think we should not provide a default for this. If a value is not provided, we should leave the url label alone.